### PR TITLE
Fix mobile: Change thread settings ui 

### PR DESCRIPTION
**Pull request by codengoo**
Mobile App: Thread Settings > It shows incorrect "Channel Settings" tittle
[#2491](https://github.com/nccasia/mezon-fe/issues/2491)

### CODE REVIEW

File: apps/mobile/src/app/components/ChannelSetting/index.tsx [hunk 1]
```diff
@@ -24,8 +24,9 @@ type ScreenChannelSetting = typeof APP_SCREEN.MENU_CHANNEL.SETTINGS;
 export default function ChannelSetting({ navigation, route }: MenuChannelScreenProps<ScreenChannelSetting>) {
 	const { themeValue } = useTheme();
 	const styles = style(themeValue);
-	const { channelId } = route.params;
+	const { channelId, isChannel } = route.params;
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/components/ChannelSetting/index.tsx]: add value isChannel default

File: apps/mobile/src/app/navigation/ScreenTypes.ts [hunk 2]
```diff
@@ -122,12 +122,13 @@ type MenuThreadStackParamList = {
...
   [APP_SCREEN.MENU_THREAD.CREATE_THREAD]: undefined;
   [APP_SCREEN.MENU_THREAD.CREATE_THREAD_FORM_MODAL]: undefined;
-  [APP_SCREEN.MENU_THREAD.MUTE_THREAD_DETAIL_CHANNEL]: { currentChannel: IChannel | DirectEntity};
+  [APP_SCREEN.MENU_THREAD.MUTE_THREAD_DETAIL_CHANNEL]: { currentChannel: IChannel | DirectEntity };
 }
 
 type MenuChannelStackParamList = {
   [APP_SCREEN.MENU_CHANNEL.SETTINGS]: {
     channelId: string;
+    isChannel: boolean;
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/navigation/ScreenTypes.ts]: isChannel?: boolean. Some screen navigate no isChannel flags needed

File: apps/mobile/src/app/screens/home/homedrawer/components/ChannelMenu/index.tsx [hunk 3]
```diff
@@ -106,7 +106,8 @@ export default function ChannelMenu({ channel, inviteRef }: IChannelMenuProps) {
 				navigation.navigate(APP_SCREEN.MENU_CHANNEL.STACK, {
 					screen: APP_SCREEN.MENU_CHANNEL.SETTINGS,
 					params: {
-						channelId: channel?.channel_id
+						channelId: channel?.channel_id,
+						isChannel: isChannel
```
Comment (nguyentrannhan) [inline apps/mobile/src/app/screens/home/homedrawer/components/ChannelMenu/index.tsx:110]: parrent_id

File: apps/mobile/src/app/navigation/index.tsx [hunk 4]
```diff
@@ -24,8 +24,10 @@ Sentry.init({
 });
 
 const mezon: CreateMezonClientOptions = {
-	host: process.env.NX_CHAT_APP_API_HOST as string,
-	key: process.env.NX_CHAT_APP_API_KEY as string,
+	// host: process.env.NX_CHAT_APP_API_HOST as string,
+	// key: process.env.NX_CHAT_APP_API_KEY as string,
+	host: "mezon.vn",
+	key: "m3zonPr0dkey",
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/navigation/index.tsx]: revert env

**Review by phuocnguyenncc — APPROVED** [review]

**Pull request merged**